templates/emails/plain/reverse-withdrawal-invoice.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Monthly Reverse Withdrawal Invoice Template.
+ *
+ * Vendors Will get this email once in a month.
+ *
+ * @class ReverseWithdrawalInvoice
+ *
+ * @package \WeDevs\Dokan\Emails\ReverseWithdrawalInvoice
+ *
+ * @since DOKAN_SINCE
+ *
+ * @var $seller_info \WeDevs\Dokan\Vendor\Vendor
+ * @var $due_status array
+ * @var $data array
+ *
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+}
+
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n";
+echo esc_html( wp_strip_all_tags( $email_heading ) );
+echo "\n=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n\n";
+
+printf( esc_attr__( 'Hi %s,\n\n', 'dokan-lite' ), esc_attr( $seller_info->get_shop_name() ) );
+
+printf( esc_attr__( 'Your %1$s %2$s invoice is now available for store: %3$s.\n\n', 'dokan-lite' ), $data['{month}'], $data['{year}'], $seller_info->get_shop_name() );
+
+printf(esc_attr__('Summary for %1$s : \n\n', 'dokan-lite'), $seller_info->get_shop_name());
+
+printf( esc_attr__( 'Reverse withdrawal charges for %1$s %2$s: %3$s \n\n', 'dokan-lite' ), $data['{month}'], $data['{year}'], wc_price( $due_status['balance']['payable_amount'] ) );
+
+printf( esc_attr__( 'Due Date %1$s : \n\n', 'dokan-lite' ), 'immediate' === $due_status['due_date'] ? ucfirst( $due_status['due_date'] ) : dokan_format_date( $due_status['due_date'] ) );
```

Comment:
`esc_html` should be used instead of all these `esc_attr`

Reply:
esc_attr is needed here, since we are not printing any HTML tags under plain text email

Reply:
By definition both `esc_html` and `esc_attr` do the same thing. The only difference is they use seperate filters.
But the guidelines say here `esc_html` should be used. `esc_attr` is meant for sanitizing the attribute values.
https://developer.wordpress.org/plugins/security/securing-output/#escaping

Reply:
good catch, I was confused with wp_strip_all_tags